PL2/PL2-core/pom.xml

```diff
@@ -26,6 +26,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
+			<version>1.9.5</version>
+		</dependency>
+        <dependency>
```

Comment:
In the GUI module I am using this dependensy for mockito:
```xml
<dependency>
            <groupId>org.mockito</groupId>
            <artifactId>mockito-core</artifactId>
            <version>1.10.19</version>
        </dependency>
</dependency>
```
@Pathemeous @justinvdk which mockito version is preferred? It would be nicer to use the same version (then travis/anyone building the project for the first time won't have to download both either).